src/Analyzer.Core.JsonRuleTests/README.md

```diff
@@ -0,0 +1,46 @@
+# Test Project for Built-in JSON Rules
```

Comment:
Wouldn't this also work for PowerShell rules? Seems like a generic framework that could potentially work with any rule type inside of the tool

Reply:
Good call.